Application/learn_static.py

```
@@ -264,34 +268,34 @@ def evaluate(self, epoch, save = True, logs = {}):
                         # we seem to have started overfitting
                         set_stop_reason("overfitting")
-                        TRex.log("[STOP] overfitting. stopping with loss diffs: "+str(change))
+                        TRex.log("[STOP] overfitting. stopping with loss diffs: "+str(change))
                         self.model.stop_training = True
 
-        self.update_status(True, logs=logs)
+        self.update_status(True, logs=logs, patience=patience)
         self.batches = 0
         return unique
     
     def on_epoch_end(self, epoch, logs={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)